pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md

```diff
@@ -271,16 +279,19 @@ To help with this, AMP will add a special attribute `amp-story-visible` to your
 Example: this animation will fire when the page comes into focus, and restart if a user clicks to another page in the story and returns.
 
 [sourcecode:html]
+
 <style amp-custom>
     body[amp-story-visible] .my-animation-class {
       animation: 2s my-animation-name;
     }
 </style>
+
 [/sourcecode]
 
 ## Sponsored Story <a name="sponsored-story"></a>
+
 A Sponsored Story exists as a URL on the web, enabling the drive of user traffic to a Sponsored Story from the call to action button on an AMP Story ad. A Sponsored Story is an AMP Story, but with focus on an immersive and expansive ad experience.
 
 {{ image('/static/img/docs/stampads/sponsored_story_full.png', 1600, 900, layout='intrinsic', alt='CTA button directs to a Sponsored Story', caption='CTA button directs to a Sponsored Story', align='' ) }}
 
-Read more about creating an [AMP Story here](amp_story_best_practices.md).
+Read more about creating an [AMP Story here](/documentation/guides-and-tutorials/start/create_successful_stories.md).
```

Comment:
Should this read "Web Story here" now?

Reply:
yes, but the whole doc needs to be updated